File: src/main/java/com/seguros/dto/ClienteRequestDTO.java
package com.seguros.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class ClienteRequestDTO {
    private Long usuarioId;
    private String tipoIdentificacion;
    private String numeroIdentificacion;
    private LocalDate fechaNacimiento;
    private String nacionalidad;
    private String estadoCivil;
    private String sexo;
    private String lugarNacimiento;
    private Double estatura;
    private Double peso;
    private String direccion;
}

File: frontGestion/src/app/models/cliente-request.dto.ts
export interface ClienteRequestDTO {
  usuarioId: number;
  tipoIdentificacion: string;
  numeroIdentificacion: string;
  fechaNacimiento: string; // Formato ISO: 'yyyy-MM-dd'
  nacionalidad: string;
  estadoCivil: string;
  sexo: string;
  lugarNacimiento: string;
  estatura: number;
  peso: number;
  direccion: string;
}
